src/app/graphs/issue-graph/issue-graph.component.ts

```diff
@@ -125,6 +131,10 @@ export class IssueGraphComponent implements OnInit, OnDestroy, AfterViewInit {
    * Cancel all subscriptions on component destruction.
    */
   ngOnDestroy() {
+    // Save the current zoom details of the graph for when the user comes back to the graph
+    localStorage.setItem(`zoomTransform_${this.projectStorageKey}`, JSON.stringify(this.graph.currentZoomTransform));
+    // Save the current bounding box of the graph for when the user comes back to the graph
+    localStorage.setItem(`zoomBoundingBox_${this.projectStorageKey}`, JSON.stringify(this.graph.currentViewWindow));
```

Comment:
There is already a json object saved for the node positions (line 45), the transform could be saved in there as well.
But this is ok for now I think.